src/polytax/train.py

```diff
@@ -173,12 +175,14 @@ def _build_model(self, train_state: "TrainState"):
         self.get("model_config")["vocab_size"] = self.tokenizer.vocab_size
 
         if "switch" in self.get("model_config/model_type"):
+            # WIP: Make these names match in the Switch Transformer config, we need to make sure all the variables match.
+            # Right now (10/12) I don't think it's consistent yet, will update my fork asap
```

Comment:
please ensure consistency and that the switch transformer can run (in at least some configuration) with the latest polytax.

Reply:
Done